test/PosthogAnalytics-test.ts

```diff
@@ -41,25 +39,9 @@ class FakePosthog {
     }
 }
 
-export interface ITestEvent extends IAnonymousEvent {
-    key: "jest_test_event";
-    properties: {
-        foo: string;
-    };
-}
-
-export interface ITestPseudonymousEvent extends IPseudonymousEvent {
-    key: "jest_test_pseudo_event";
-    properties: {
-        foo: string;
-    };
-}
-
-export interface ITestRoomEvent extends IRoomEvent {
-    key: "jest_test_room_event";
-    properties: {
-        foo: string;
-    };
+export interface ITestEvent extends IEvent {
+    eventName: "jest_test_event";
```

Comment:
We should follow our own docs in tests too 😇
```suggestion
    eventName: "JestTestEvent";
```